test/replication-luatest/election_strict_fencing_test.lua

```diff
@@ -0,0 +1,108 @@
+local luatest = require('luatest')
+local cluster = require('test.luatest_helpers.cluster')
+local server = require('test.luatest_helpers.server')
+local g = luatest.group('fencing_mode', {
+    {election_fencing_mode = 'soft'}, {election_fencing_mode = 'strict'}})
```

Comment:
The fact that you run with `soft` mode too doesn't really look good with the file name which says "strict_fencing". Given that you also copied notable amount of code from the original `election_fencing_test.lua`, I would suggest to move the present test into that file.

Reply:
I've moved both test to election_fencing_test.lua, in new group.